contracts/sfc/SFC.sol

```diff
@@ -496,65 +518,80 @@ contract SFC is Initializable, Ownable, Version {
         emit RestakedRewards(delegator, toValidatorID, rewards);
     }
 
+    // Get the current epoch number
     function currentEpoch() public view returns (uint256) {
         return currentSealedEpoch + 1;
     }
 
+    // Get self-stake of a validator
     function getSelfStake(uint256 validatorID) public view returns (uint256) {
         return getStake[getValidator[validatorID].auth][validatorID];
     }
 
+    // Get validator IDs for given epoch
     function getEpochValidatorIDs(uint256 epoch) public view returns (uint256[] memory) {
         return getEpochSnapshot[epoch].validatorIDs;
     }
 
+    // Get received stake for a validator in a given epoch
     function getEpochReceivedStake(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].receivedStake[validatorID];
     }
 
+    // Get accumulated reward per token for a validator in a given epoch
     function getEpochAccumulatedRewardPerToken(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].accumulatedRewardPerToken[validatorID];
     }
 
+    // Get accumulated uptime for a validator in a given epoch
     function getEpochAccumulatedUptime(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].accumulatedUptime[validatorID];
     }
 
+    // Get accumulated originated txs fee for a validator in a given epoch
     function getEpochAccumulatedOriginatedTxsFee(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].accumulatedOriginatedTxsFee[validatorID];
     }
 
+    // Get offline time for a validator in a given epoch
     function getEpochOfflineTime(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].offlineTime[validatorID];
     }
 
+    // Get offline blocks for a validator in a given epoch
     function getEpochOfflineBlocks(uint256 epoch, uint256 validatorID) public view returns (uint256) {
         return getEpochSnapshot[epoch].offlineBlocks[validatorID];
     }
 
+    // Get end block for a given epoch
     function getEpochEndBlock(uint256 epoch) public view returns (uint256) {
         return getEpochSnapshot[epoch].endBlock;
     }
 
+    // Get slashed status for a validator
     function isSlashed(uint256 validatorID) public view returns (bool) {
         return getValidator[validatorID].status & CHEATER_MASK != 0;
     }
 
+    // Get pending rewards for given delegator and validator
```

Comment:
Get the amount of rewards which can be currently claimed by the given delegator for the given validator.